charts/replicated-library/README.md

```diff
@@ -70,6 +70,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 #### Added
 
 - Make RBAC for preflights optional
+- Add containers.env capability to leverage maps (ie: secretRef / configMapRef / name-value)
```

Comment:
also this message here isn't the clearest in terms of what's being added. I think it could be something more along the lines of:

> Adding support for native Kubernetes syntax for container environment variables. Supported formats are now:
....

Reply:
add messaging as requested.